soc/arm64/ti_sitara/am6x/mmu_regions.c

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2023 Enphase Energy
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/arch/arm64/arm_mmu.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/sys/util.h>
+
+/* Only 4K granule is supported */
+#define MMU_REGION_MIN_SIZE KB(4)
+
+static const struct arm_mmu_region mmu_regions[] = {
+
+	MMU_REGION_FLAT_ENTRY("GIC",
+			      DT_REG_ADDR_BY_IDX(DT_NODELABEL(gic), 0),
+			      DT_REG_SIZE_BY_IDX(DT_NODELABEL(gic), 0),
+			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
+
+	MMU_REGION_FLAT_ENTRY("GIC",
+			      DT_REG_ADDR_BY_IDX(DT_NODELABEL(gic), 1),
+			      DT_REG_SIZE_BY_IDX(DT_NODELABEL(gic), 1),
+			      MT_DEVICE_nGnRnE | MT_P_RW_U_NA | MT_NS),
+
+	MMU_REGION_FLAT_ENTRY("UART0",
```

Comment:
Do we need to add `UART` items into `mmu_regions[]`?
The `uart_ns16550` driver can map the mmio address by itself.
https://github.com/zephyrproject-rtos/zephyr/blob/63850804f167353df27b6b33d05f8f3ab642aae4/drivers/serial/uart_ns16550.c#L350

Reply:
I wasn't aware of that, I will remove it (this actually explains a lot...)